src/emoji.ts

```diff
@@ -15,26 +15,23 @@ limitations under the License.
 */
 
 import EMOJIBASE from 'emojibase-data/en/compact.json';
+import SHORTCODES from 'emojibase-data/en/shortcodes/iamcal.json';
 
 export interface IEmoji {
     annotation: string;
-    group: number;
+    group?: number;
     hexcode: string;
-    order: number;
+    order?: number;
     shortcodes: string[];
-    tags: string[];
+    tags?: string[];
     unicode: string;
+    skins?: Omit<IEmoji, "shortcodes">[]; // Currently unused
```

Comment:
I don't think it has `tags` either, correct me if I'm wrong

Reply:
It typechecks, so you must be right

Reply:
Well sub-types are always allowed as receivers so it'd typecheck fine either way